k8s/cloud/base/api_deployment.yaml

```diff
@@ -135,6 +135,20 @@ spec:
           mountPath: /vizier-image-secret
         - name: elastic-certs-pub
           mountPath: /elastic-certs-pub
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - ALL
+          runAsNonRoot: true
+          runAsUser: 10100
```

Comment:
Is there any reason why you chose `10100` rather than `1000` which is more commonly used?

Reply:
It's recommends to have higher UIDs (10000+) for containers won't overlap with host system's 
- https://docs.bridgecrew.io/docs/bc_k8s_37
- https://hub.datree.io/built-in-rules/prevent-uid-conflicts